Tests/ApolloCodegenTests/Frontend/DocumentParsingAndValidationTests.swift

```diff
@@ -0,0 +1,165 @@
+import XCTest
+import ApolloTestSupport
+@testable import ApolloCodegenLib
+
+class DocumentParsingAndValidationTests: XCTestCase {
+  
+  var codegenFrontend: ApolloCodegenFrontend!
+  var schema: GraphQLSchema!
+  
+  override func setUpWithError() throws {
```

Comment:
I know this is good practice in general and I should add this. But it is merely ceremonial because the superclass implementation doesn't do anything. The code Xcode generates, or Apple's sample code, never calls super either.

Reply:
For what it's worth that's because [the default implementation is currently empty](https://stackoverflow.com/a/53455661/681493). Honestly, I'd rather not rely on that remaining the case, and I'm pretty surprised Apple is to the extent that they've changed the generated code.